extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -59,16 +59,16 @@
 		<div class="masthead-info-lower">
 			<div class="contributions-details tally">
 				<? if ( !empty( $user['registration'] ) ): ?>
-					<a href="<?= Sanitizer::encodeAttribute( $user['contributionsURL'] ) ?>">
-						<em><?= $user['edits'] ?></em>
+					<a href="<?= Sanitizer::encodeAttribute( $user['contributionsURL'] ); ?>">
+						<em><?= htmlspecialchars( $user['edits'] ); ?></em>
 						<span>
-							<?= wfMessage( 'user-identity-box-edits-since-joining', $user['registration'] )->plain() ?>
+							<?= wfMessage( 'user-identity-box-edits-since-joining' )->rawParams( $user['registration'] )->parse(); ?>
```

Comment:
Registration doesn't need to be in `rawParams` (and probably shouldn't since it has no pre-escaped HTML in it).